### DISCUSSION TITLE
Support to resume reading

### DISCUSSION THREAD

**Discussion by xeivier** [RaRe-Technologies/smart_open#626] [Q&A]
Is there a way to start reading where the last iteration finished? I mean, storing the buffer somehow (it could be up to the lib user to decide where to store it and then provide it back to the lib in the next iteration) and the start and end byte of the last iteration to resume reading after. Just to clarify, I mean resuming reading in another request, not the same that started reading.

**Comment by mpenkov** [comment]
Can you not use .tell and .seek to achieve this?

https://docs.python.org/3/library/io.html#io.IOBase.seek

**Reply by xeivier** [reply]
From what I know, the only exposed API goes this way:
# stream from GCS
for line in open('gs://my_bucket/my_file.txt'):
    print(line)

How IOBase.seek fit in this?

Thanks in advance
